Template/LO/SubProcesses/reweight.f

```diff
@@ -633,6 +633,17 @@ logical function setclscales(p, keepq2bck, ivec)
       external is_octet
       setclscales=.true.
 
+       setclscales=.true.
+
+c Workaround for valgrind 'Conditional jump or move depends on uninitialised value(s)'
+c See https://github.com/mg5amcnlo/mg5amcnlo/issues/111
+c FIXME: this is just a workaround to avoid uninitialised values and undefined behaviour...
+c FIXME: a real bug is probably hidden in the code (goodjet must be correctly defined!)...
+c FIXME: adding this workaround may change the behaviour of existing code...
+      do i=1,n_max_cl
+        goodjet(i)=.false. !!FIXME!! there is no reason to choose false instead of true here...
+      end do
+
```

Comment:
```suggestion
     setclscales=.true.
c    ensure that goodjet start on False to avoid potential undefined behavior since the algorithm           
c    does call unset variable. The associated behavior was detected stable due to the presence of the 
c   requirement of other condition in the same if statement. In case of weird behavior, it is adivsed to add 
c   a second array to check which element are set from those which are not      
         goodjet(:) = .false.
```